build.gradle

```diff
@@ -305,7 +305,11 @@ repositories {
 
 dependencies {
 
-    implementation group: 'javax.validation', name: 'validation-api', version: '2.0.1.Final'
+    testImplementation ('com.github.hmcts:rd-commons-lib:v0.0.9'){
```

Comment:
Hi Amit we can see that there is another dependency of rd-commons-lib of version v0.0.8 can you please remove that and remove the serenity dependencies as well ,as that is the key part of this ticket ....

Reply:
removed and it is working well.